nextest-runner/src/reuse_build/archiver.rs

```diff
@@ -295,6 +316,46 @@ impl<'a, W: Write> Archiver<'a, W> {
         Ok(())
     }
 
+    fn append_path_recursive(
+        &mut self,
+        rel_path: &Utf8Path,
+        src_path: &Utf8Path,
+    ) -> Result<(), ArchiveCreateError> {
+        let entries =
+            src_path
+                .read_dir_utf8()
+                .map_err(|error| ArchiveCreateError::InputFileRead {
```

Comment:
We should indicate here that the file read is user-specified and not something that nextest archives by default.

Reply:
Will do this in a followup.